### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by malshebo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: We can replace this with just OS (Linux, Mac, Windows)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Not needed I dont think. Nobody will be running scripts on their phone :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: I think this sentence got cut off

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Why no title?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+labels: ''
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Auto assign the Bug label please

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
+
+**Additional context**
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: Might be nice to add in another section for possible solutions to fix the bug

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: I think this should be written in yaml, like the data and documentation templates?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Can we add in a section about what conda environment they are using. Or make it clearer in the To Reporduce section that that info needs to be included, please

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    description: How do you trigger this bug? Please walk us through it step by step.
+    placeholder: |
+      1. Running this command...
+      2. Produces this result...
+  validations:
+    required: false
+
+- type: input
+  attributes:
+    label: Operating System
```
Comment (trevorb1) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: Can we change this one to a dropdown? I feel like knowing the exact version of MacOS or whatever isnt going to be where I start debugging haha. Maybe something like 

```yaml 
  - type: dropdown
    id: os
    attributes:
      label: Operating System
      description: What operating system are you running?
      options:
        - Linux
        - MacOS
        - Windows
    validations:
      required: true
```

**Review by trevorb1 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 9 above)
Comment (malshebo) [reply .github/ISSUE_TEMPLATE/bug_report.yml:48]: Yep can do!

**Review by trevorb1 — APPROVED** [review]
Thanks!

**Pull request merged**